packages/salesforcedx-vscode-core/src/commands/forceSourcePullLegacy.ts

```diff
@@ -0,0 +1,60 @@
+/*
+ * Copyright (c) 2017, salesforce.com, inc.
+ * All rights reserved.
+ * Licensed under the BSD 3-Clause license.
+ * For full license text, see LICENSE.txt file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+
+import {
+  Command,
+  SfdxCommandBuilder
+} from '@salesforce/salesforcedx-utils-vscode/out/src/cli';
+import { nls } from '../messages';
+import {
+  EmptyParametersGatherer,
+  FlagParameter,
+  SfdxCommandlet,
+  SfdxCommandletExecutor,
+  SfdxWorkspaceChecker
+} from './util';
+export class ForceSourcePullLegacyExecutor extends SfdxCommandletExecutor<{}> {
+  private flag: string | undefined;
+
+  public constructor(flag?: string) {
+    super();
+    this.flag = flag;
+  }
+
+  public build(data: {}): Command {
+    const builder = new SfdxCommandBuilder()
+      .withDescription(
+        nls.localize('force_source_pull_legacy_default_scratch_org_text')
+      )
+      .withArg('force:source:pull')
+      .withLogName('force_source_pull_legacy_default_scratch_org');
```

Comment:
it'd be cool to pull these string up into consts in the file.

Reply:
done